.github/workflows/main.yml

```
@@ -22,11 +22,11 @@ jobs:
         python -m pip install --upgrade pip
         pip install .
     - name: Create Examples
-      run: PYTHONPATH=$(pwd)/src:$PYTHONPATH cd src/wireviz/ && python build_examples.py
+      run: PYTHONPATH=$(pwd)/src:$PYTHONPATH && python src/wireviz/tools/build_examples.py
```

When removing the `cd` command, why still keep the `&&` in front of the `python` command?

---

If it works without, throw it out :)

---

How do you suggest I verify that this change will work?
```suggestion
      run: PYTHONPATH=$(pwd)/src:$PYTHONPATH python src/wireviz/tools/build_examples.py
```
Will the modified code be executed immediately after adding it as a commit to this PR branch, or does it have to be merged into the master branch to be executed?

---

good question. Does every branch/PR run the workflows defined within itself? One test could be to remove the workflow entirely, or insert a call to a nonexistent module, and see after committing and pushing the change, the checks stop passing. If so, verifying the effect of removing `&&` should be straightforward. If not, leave it as-is for now and test later, e.g. in a test repo.

---

> Will the modified code be executed immediately after adding it as a commit to this PR branch, or does it have to be merged into the master branch to be executed?

Tested, yes, the change is immediately applied. Removed `&&`, build still works. Resolved.